src/rpc.ts

```diff
@@ -3,11 +3,25 @@
 
 export interface ICallable {
   exec(service: keyof typeof Services, method: string, params: any): Promise<any>;
+  getConnection(service: keyof typeof Services): Promise<any>;
```

Comment:
Not without introducing a ton of type assertions -- we'd need to essentially have a map of strings to a union of several different function types and then assert in every single method we call. Which in the end is no safer than `any`. The only consumer of this is the autogenerated classes -- the autogenerated clients _are_ the narrowed typings. The primitive exec/getConnection calls are not exposed to consumers of the package.